README.md

```diff
@@ -5,9 +5,15 @@ This website is built using [Docusaurus 2](https://v2.docusaurus.io/), a modern
 ## Installation
 
 ```console
+brew install npm # skip if you have node installed already
```

Comment:
brew is not a standard way to install on all platforms, it's clearer to say "install node and yarn on your device", or something akin to that.